packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -158,6 +158,14 @@ const VirtualListBaseFactory = (type) => {
 			 */
 			getComponentProps: PropTypes.func,
 
+			/**
+			 * It is the object which is passed as arguments of `itemsRenderer` prop function.
+			 *
+			 * @type {Object}
+			 * @private
+			 */
+			itemProps: PropTypes.object,
```

Comment:
I'd like to use `childProps` rather than `itemProps`. It's a change I've wanted to make to `Repeater` and `Group` for while after we decided to standardize on `childComponent`. It avoids using `child` and `item` to refer to the same thing.

Also, I don't think you need to mark this as private. It's a stable concept that we've already demonstrated in `Repeater`.